packages/aragon-wrapper/src/cache/index.js

```diff
@@ -54,6 +54,19 @@ export default class Cache {
     return value || defaultValue
   }
 
+  async remove (key) {
+    await this.db.removeItem(key)
+    this.changes.next({ key, value: null })
+  }
+
+  async clear () {
+    await this.db.clear()
+
+    // Reset the changes observable, since we've cleared the entire cache
+    this.changes.complete()
+    this.changes = new Subject()
```

Comment:
Curious what you guys think... this seems like the most obvious way to tell a user we're basically restarting the cache from scratch, but would also mean users would have to know about this and be re-subscribed.

Nothing is actually attached to these changes for now so it's not a big problem (perhaps this is observable is something we should just altogether remove?)

Reply:
I understand your concern. Seems reasonable to close the observable. 

+1 for removing this observable. Can we think of a reasonable use case for observing cache changing over time? 

This follows from the logic of using cache to improve performance ad hoc per request rather than per data change.

Reply:
Actually I lied; it's being used in `@aragon/client`: https://github.com/aragon/aragon.js/blob/master/packages/aragon-client/src/index.js#L180, and this is used in all the apps' frontends to subscribe to state changes in the apps.

Given that this is actually only observing a specific key, I think it actually makes more sense if we just send a `null` value back and not reset the observable. Moreover, `@aragon/client` subscriptions to this event would never know about the observable finishing in `@aragon/wrapper`'s context, since we don't transmit an end event over the `postMessage`.

Reply:
Fair enough. Yeah in that case we probably don't want to break this api.

Given the layers of abstraction and isolation in place, what do you think about cache being hidden away behind a simpler state call/subscription?

Reply:
> What do you think about cache being hidden away behind a simpler state call/subscription?

On some level this is already done for the apps (https://github.com/aragon/aragon.js/blob/master/packages/aragon-client/src/index.js#L180 returns an observable that emits on cache changes).

`aragon/aragon` doesn't use the cache very often, and `@aragon/wrapper` exposes it directly, so there's not much need for another abstraction.